### ISSUE TITLE
Equivalencing namespaces by string match

### ISSUE THREAD

**Issue by wshayes** [belbio/bel_resources#45]
From Natalie
Looking here - https://github.com/OpenBEL/resource-generator/blob/master/rdf.py -  in the resource generator code, it appears that I was only building the string match equivalences for abundances and biological processes. The function used was “get_close_matches”. We used a different skos relationship (closeMatch) for these than for the other equivlences (exactMatch, except when it was not a 1:1).

Abundance namespaces (ChEBI and ChEMBL)  have equivalences (ChEMBL -> ChEBI and from both to INCHI_KEY).

Biological processes need to be equivalenced - the old OpenBEL RDF approach was to use lowercased names/synonyms and do exact string-match on them.

**Comment by ncatlett** [comment]
Are you bringing in any chemicals (abundances) via MESH? I think these were the reason we needed the string match for equivalencing.

**Comment by wshayes** [comment]
Good point - I am.  tools/namespaces/mesh.py also needs to have equivalences processed for abundances.
